data_module/datamodule.py

```diff
@@ -1,26 +1,155 @@
 import dataset
-from torch.utils.data import DataLoader
+import torch
+from torch.utils.data import Dataset, DataLoader, RandomSampler
 import pandas as pd
 import lightning as L
+from data_preprocessor import TextPreprocessing
+from transformers import AutoModelForSeq2SeqLM, AutoTokenizer
+device = "cuda" if torch.cuda.is_available() else "cpu"
+
+
+
+class TextSummaryDataset(Dataset):
+    def __init__(self,
+                 df,
+                 textprocessor,
+                 tokenizer,
+                 tokenizer_chapter_max_length=1024,
+                 tokenizer_summary_max_length=64,
+                 truncation=True,
+                 ):
```

Comment:
@shivanshvermaa  Move class TextSummaryDataset() to dataset.py, make changes, and push on the same PR